docs/authorization/policies.md

```diff
@@ -158,25 +203,25 @@ These privileges are for DataHub operators to access & manage the administrative
-| Manage Business Attribute       | Allow actor to create, update, delete Business Attribute                                                           |
-| Manage Documentation Forms      | Allow actor to manage forms assigned to assets to assist in documentation efforts.                                 |
-| Manage Metadata Ingestion       | Allow actor to create, remove, and update Metadata Ingestion sources.                                              |
-| Manage Features                 | Umbrella privilege to manage all features.                                                                         |
-| View Analytics                  | Allow actor to view the DataHub analytics dashboard.                                                               |
-| Manage Public Views             | Allow actor to create, update, and delete any Public (shared) Views.                                               |
-| Manage Ownership Types          | Allow actor to create, update and delete Ownership Types.                                                          |
-| Create Business Attribute       | Allow actor to create new Business Attribute.                                                                      |
-| Manage Structured Properties    | Manage structured properties in your instance.                                                                     |
-| View Tests                      | View Asset Tests.                                                                                                  |
-| Manage Tests[^1]                | Allow actor to create and remove Asset Tests.                                                                      |
-| View Metadata Proposals[^1]     | Allow actor to view the requests tab for viewing metadata proposals.                                               |
-| Create metadata constraints[^2] | Allow actor to create metadata constraints.                                                                        |
-| Manage Platform Settings[^1]    | Allow actor to view and change platform-level settings, like integrations & notifications.                         |
-| Manage Monitors[^1]             | Allow actor to create, update, and delete any data asset monitors, including Custom SQL monitors. Grant with care. |
-| View Manage Tags                | Allow the actor to view the Manage Tags page.                                                                      |
+| Platform Privileges             | Description                                                                                                               |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| Manage All Home Page Posts      | Allow actor to create and delete home page posts. This corresponds to the 'post' resource type.                           |
```

Comment:
This shouldn't specify all, it's configurable on the policy whether or not it applies to specific entities.